tests/fixtures/models/store.py

```diff
@@ -15,14 +16,174 @@
 
 import pytest
 
+from ...utils import update_store, create_store
+
+
+DEFAULT_STORE_UNITS_1 = [("Unit 1", "Unit 1"),
+                         ("Unit 2", "Unit 2")]
+
+DEFAULT_STORE_UNITS_2 = [("Unit 3", "Unit 3"),
+                         ("Unit 4", "Unit 4"),
+                         ("Unit 5", "Unit 5")]
+
+DEFAULT_STORE_UNITS_3 = [("Unit 6", "Unit 6"),
+                         ("Unit 7", "Unit 7"),
+                         ("Unit 8", "Unit 8")]
+
+UPDATED_STORE_UNITS_1 = [(src, "UPDATED %s" % target)
+                         for src, target
+                         in DEFAULT_STORE_UNITS_1]
+
+UPDATED_STORE_UNITS_2 = [(src, "UPDATED %s" % target)
+                         for src, target
+                         in DEFAULT_STORE_UNITS_2]
+
+UPDATED_STORE_UNITS_3 = [(src, "UPDATED %s" % target)
+                         for src, target
+                         in DEFAULT_STORE_UNITS_3]
+
 
 TEST_UPDATE_PO = "tests/data/po/tutorial/en/tutorial_update.po"
 TEST_EVIL_UPDATE_PO = "tests/data/po/tutorial/en/tutorial_update_evil.po"
 
+UPDATE_STORE_TESTS = OrderedDict()
+UPDATE_STORE_TESTS['min_empty'] = {"update_store": (0, [])}
+UPDATE_STORE_TESTS['min_new_units'] = {
+    "update_store": (0, DEFAULT_STORE_UNITS_3)
```

Comment:
Nit: always use a trailing comma — it helps reduce future diff noise. Same elsewhere.